DistributedLock.Tests/AbstractTestCases/DistributedLockCoreTestCases.cs

```diff
@@ -443,7 +443,7 @@ private void CrossProcessAbandonmentHelper(bool asyncWait, bool kill)
             }
         }
 
-        private Command RunLockTaker(TLockProvider engine, params string[] args)
+        private Command RunLockTaker(TLockProvider engine, string lockType, string lockName)
```

Comment:
Why was this change needed?

Reply:
@madelson  The need to run `RunLockTaker` / `DistributedLockTaker` with global lock name (prefix + local name) in order to pass cross process tests (`DistributedLockTaker` know nothing about lock prefix, need to pass it somehow).